src/KubernetesClient/KubernetesClient.csproj

```diff
@@ -7,7 +7,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="7.0.0" />
```

Comment:
so this is net6 only?

we already moved to net8 lol

Reply:
Yeah, the `System.Diagnostics.DiagnosticSource` is included in the shared framework, so there's no need to reference this package for most cases.

While we're using the `UpDownCounter` in the code

https://github.com/kubernetes-client/csharp/blob/2c2bd6a4a388f630f9aac84cbe4b8a65f0096ee1/src/KubernetesClient/PrometheusHandler.cs#L33

The `UpDownCounter` has been introduced since .NET 7
see the API: https://apisof.net/catalog/c7f52491f5c19b75d271677fe9123886

The shared framework for .NET 6 only has the 6.0.x reference which does not support this `UpDownCounter`, so we had to add a reference explicitly.

For .NET 7 and later frameworks, we could use the framework reference directly, so no need to add the reference for `System.Diagnostics.DiagnosticSource` when targeting `net8.0` I think